src/pages/workspace/tags/WorkspaceViewTagsPage.tsx

```diff
@@ -121,11 +129,18 @@ function WorkspaceViewTagsPage({route}: WorkspaceViewTagsProps) {
                             isOn={tag.enabled}
                             disabled={tag.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE}
                             accessibilityLabel={translate('workspace.tags.enableTag')}
-                            onToggle={(newValue: boolean) => updateWorkspaceTagEnabled(newValue, tag.name)}
+                            onToggle={(newValue: boolean) => {
+                                if (getCountOfMultiLevelRequired(policyTags) === 1 && getCountOfEnabledTagsOfList(currentPolicyTag?.tags) === 1 && policy?.requiresTag && tag.enabled) {
+                                    setShowCannotDisableLastTagModal(true);
+                                    return;
+                                }
+                                updateWorkspaceTagEnabled(newValue, tag.name);
+                            }}
+                            showLockIcon={getCountOfMultiLevelRequired(policyTags) === 1 && getCountOfEnabledTagsOfList(currentPolicyTag?.tags) === 1 && policy?.requiresTag && tag.enabled}
                         />
```

Comment:
The nested tags logic to be able to be disabled or not should be based on the tag list being required or not. IOW do not use `policy?.requiresTag`. Even if `policy?.requiresTag` is false a tag shouldn't be disabled if it's the last one in the list (and that list is required).